src/lib.rs

```diff
@@ -90,6 +90,7 @@ impl Default for TextBounds {
 }
 
 /// A text area containing text to be rendered along with its overflow behavior.
+#[derive(Clone, Copy)]
```

Comment:
`Clone` seems ok but could we leave off `Copy` for now? It seems likely that this API could change with future changes to cosmic-text's attributes, so we might not be able to keep it as `Copy` later